impl/src/main/java/com.silviatanas.project.contract.impl/ContractManagementImpl.java

```diff
@@ -0,0 +1,47 @@
+package com.silviatanas.project.contract.impl;
+
+import com.silviatanas.project.contract.api.Contract;
+import com.silviatanas.project.contract.api.ContractManagement;
+import com.silviatanas.project.contract.spi.ContractStorage;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.ServiceLoader;
+
+/**
+ * Implementation for Contract management methods
+ */
+public class ContractManagementImpl implements ContractManagement {
+    ServiceLoader<ContractStorage> serviceLoader = ServiceLoader.load(ContractStorage.class);
+    ContractStorage storage = serviceLoader.iterator().next();
```

Comment:
What if the service is not loaded (serviceLoader.iterator().next() return null)? 
Log storage instance with severity info. it is good practice to know real instance of the dynamic resources.